### PR TITLE
feat: more readme improvements

### PR DESCRIPTION
**Pull request by mihai-sysbio**
### Main improvements in this PR:

This PR brings some improvements related to #221, mainly by replacing the readthedocs setup with GitHub Pages. The README has been slightly restructured, by combining the Dependency section with the installation of said dependencies. 

With this PR, the automatic creation of GH Pages for the root directory of the `main` has been turned on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Due to significant refactoring of the code, GECKO version 3 is largely not backwards compatible with earlier GECKO versions.
+
+- Most notably, GECKO 3 ecModels have an `.ec` structure containing all enzyme and kcat information.
+- In addition, in GECKO 3 enzymes are incorporated in the S-matrix as MW/kcat, while in GECKO 1 and 2 this was 1/kcat (where the MW was instead considered in the protein exchange reactions).
+- GECKO 3 ecModels can be stored in YAML file format that retains all model content.
+- Most functions in GECKO 3 do not work on ecModels generated with GECKO versions 1 or 2.
+- ecModels generated in GECKO 3 do not work with functions from GECKO versions 1 or 2.
+- At this moment, there are no Python functions to work with GECKO 3 formatted ecModels.
+- The last GECKO 2 release (2.0.3) is available `here <https://github.com/SysBioChalmers/GECKO/releases/tag/v2.0.3>`.
```
Comment (mihai-sysbio) [inline README.md:66]:
```suggestion
- The last GECKO 2 release (2.0.3) is available [here](https://github.com/SysBioChalmers/GECKO/releases/tag/v2.0.3).
```

**Review by edkerk — APPROVED** [review]
Looks good, can we sneak peek the GitHub pages already?

**Comment by mihai-sysbio** [comment]
> Looks good, can we sneak peek the GitHub pages already?

I've changed the `main` branch to this one in the Pages settings, so that should be reverted when this gets merged. This is the landing page: https://sysbiochalmers.github.io/GECKO/

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- For GECKO release 2, please cite [Domenzain et al. (2022) https://doi.org/10.1038/s41467-022-31421-1](https://doi.org/10.1038/s41467-022-31421-1).
+- For GECKO release 1, please cite [Sánchez et al. (2017) https://doi.org/10.15252/msb.20167411](https://doi.org/10.15252/msb.20167411).
+
+### Getting started
+
+#### Required software
+
+- MATLAB version 2019b or later, no additional MathWorks toolboxes are required.
+- [RAVEN Toolbox](https://github.com/SysBioChalmers/RAVEN) version 2.7.12 or later. The RAVEN Toolbox Wiki contains [installation instructions for both RAVEN](https://github.com/SysBioChalmers/RAVEN/wiki/Installation) and [Gurobi](https://github.com/SysBioChalmers/RAVEN/wiki/Installation#solvers). Briefly, RAVEN is either downloaded via `git clone`, as ZIP-archive from GitHub, or installed as a [MATLAB AddOn](https://se.mathworks.com/matlabcentral/fileexchange/112330-raven-toolbox). After finishing all installation instructions, the user should run installation checks in MATLAB with: `checkInstallation`.
+
```
Comment (mihai-sysbio) [inline README.md:25]:
```suggestion
```

**Comment by mihai-sysbio** [comment]
It this ready do be merged? And if so, do we want it to go `develop` first?

**Comment by edkerk** [comment]
Yes & yes, so that `develop` and `main` are in sync.

**Pull request merged**